tests/parse_and_replace.rs

```diff
@@ -13,9 +13,9 @@ extern crate difference;
 
 use std::collections::HashSet;
 use std::ffi::OsString;
+use std::fs;
 use std::path::{Path, PathBuf};
 use std::process::Output;
-use std::fs;
```

Comment:
cargo fmt I think, I guess it doesn't normally format tests?

Reply:
Yeah, I think you can pass `--all-targets` or something the cargo-fmt